Add description about the common types of breadcrumbs (#1592)
magento/merchdocs: src/catalog/navigation-breadcrumb-trail.md

![]({% link images/images/storefront-breadcrumb-trail.png %}){: .zoom}
_Breadcrumb Trail_

## General types of bread crumbs

Breadcrumbs can be divided into three main types, which differ in their purpose. The essence and main principles of the implementation of each type of bread crumbs are described below.

### Hierarchy-based breadcrumbs

> This kind of breadcrumbs are based on the category hierarchy set up on the site. The chains presented tell the user at what structural level they are. In this case, each text link is intended for a page that is one level higher than the previous one.

Reviewer: ```suggestion
This type of breadcrumb is based on the category hierarchy set up on the site. The chains presented tell the user where in the structure they are. In this case, each text link is intended for a page that is one level higher than the previous one.
```

Example: `Men > Tops > Hoodies & Sweatshirts`

> This type of breadcrumb helps users easily see which category level they are in and provide easy navigation between catalog pages.

Reviewer: ```suggestion
The advantage of this type is that users can easily see which category level they are in and have easy access to navigation between catalog pages.
```

### History-based breadcrumbs

History (or path) -based navigation is similar to the back button in a browser. This allows users to quickly return to the previous pages they have visited without changes.

> This type of breadcrumbs is useful when customers want to return to a previous page after selecting multiple filters on a category page.

Reviewer: ```suggestion
The advantage of this type is that it is most helpful when customers want to return to a previous page after selecting multiple filters on a category page.
```

Example: `Home > What's New > Gear > Bags`

### Attribute-based breadcrumbs

> This type of angry crumbs displays the attributes selected on the category page. The main difference from the previous types is that the attribute-based breadcrumbs represent the filters and options you have selected in the navigation layer for certain products (price, quality, color, etc.).

Reviewer: ```suggestion
This type of breadcrumb displays the attributes selected on the category page. The main difference from the other types is that the attribute-based breadcrumbs represent the filters and options you have selected in the navigation layer for certain products (such as price, quality, and color).
```
